petpals/templates/forum.html

```diff
@@ -138,37 +129,37 @@ <h1>Forum</h1>
 
             <!-- Forum List -->
             <div class="forum-main-body p-2 p-sm-3 collapse forum-content show">
-              <!--First Entry-->
+              {% for post, user in posts %}
```

Comment:
Gives `TypeError: cannot unpack non-iterable Post object`. Because posts is a list of posts, not a zip of posts and users.